src/import_map.json

```diff
@@ -45,9 +45,12 @@
     "https://deno.land/std@0.161.0/": "https://deno.land/std@0.166.0/",
     "https://deno.land/std@0.101.0/": "https://deno.land/std@0.166.0/",
     "https://deno.land/std@0.105.0/": "https://deno.land/std@0.166.0/",
-    "https://deno.land/std@0.91.0/": "https://deno.land/std@0.166.0/",
     "https://deno.land/std@0.93.0/": "https://deno.land/std@0.166.0/",
 
+    "https://deno.land/std@0.91.0/path/": "https://deno.land/std@0.166.0/path/",
+    "https://deno.land/std@0.91.0/fs/": "https://deno.land/std@0.166.0/fs/",
+
+    "https://deno.land/std@0.91.0/hash/": "./resources/vendor/deno-land/std@0-91-0/hash/",
```

Comment:
This split solves our vendoring troubles. It's not a general solution but it gets puppeteer to see the right version of the needed files.